packages/amesos2/CMakeLists.txt

```diff
@@ -107,36 +103,6 @@ TRIBITS_ADD_OPTION_AND_DEFINE(${PACKAGE_NAME}_ENABLE_ShyLU_NodeBasker
   ${Amesos2_HAVE_SHYLUBASKER}
 )
 
-TRIBITS_ADD_OPTION_AND_DEFINE(${PACKAGE_NAME}_ENABLE_MUMPS
-  HAVE_AMESOS2_MUMPS
-  "Enable MUMPS in Amesos2"
-  OFF
-)
-
-ASSERT_DEFINED(TPL_ENABLE_METIS)
-TRIBITS_ADD_OPTION_AND_DEFINE(${PACKAGE_NAME}_ENABLE_METIS
-  HAVE_AMESOS2_METIS
-  "Enable METIS in Amesos2"
-  "${TPL_ENABLE_METIS}"
-  )
-ASSERT_DEFINED(${PACKAGE_NAME}_ENABLE_METIS)
-
-ASSERT_DEFINED(TPL_ENABLE_CUSPARSE)
-TRIBITS_ADD_OPTION_AND_DEFINE(${PACKAGE_NAME}_ENABLE_CUSPARSE
-  HAVE_AMESOS2_CUSPARSE
-  "Enable cuSPARSE in Amesos2"
-  "${TPL_ENABLE_CUSPARSE}"
-  )
-ASSERT_DEFINED(${PACKAGE_NAME}_ENABLE_CUSPARSE)
```

Comment:
No need for for this. TriBITS already defines these vars automatically. See:

* https://tribits.org/doc/TribitsDevelopersGuide.html#how-to-add-a-new-tribits-tpl-dependency